samples/dashboards/demo/climate/demo.js

```diff
@@ -1038,69 +1035,56 @@ async function updateBoard(board, city, column, scale, newData) {
     }
 
     const rangeTable = selectionTable.modified;
+    const rangeEnd = rangeTable.getRowCount() - 1;
 
     // Update world map
+    const mapPoints = worldMap.chart.series[1].data;
+    const lastTime = rangeTable.getCellAsNumber('time', rangeEnd);
+
+    for (const mapPoint of mapPoints) {
```

Comment:
I checked it and it turned out that the regular loop is 2x faster then for/of.
- https://jsfiddle.net/mxcfyvkw/2/
(The newest chrome).

I think that in the old browsers or CPU, the difference will be more significant.

Reply:
Good (map)point. 👍